gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/ApiMapper.java

```diff
@@ -116,6 +129,17 @@ default List<Api> map(List<GenericApiEntity> apiEntities, UriInfo uriInfo, Funct
     @Mapping(target = "links", expression = "java(computeApiLinks(apiEntity, uriInfo))")
     ApiV4 mapToV4(ApiEntity apiEntity, UriInfo uriInfo, GenericApi.DeploymentStateEnum deploymentState);
 
+    @Mapping(target = "apiVersion", source = "source.version")
+    @Mapping(target = "analytics", source = "source.apiDefinitionV4.analytics")
+    @Mapping(target = "endpointGroups", source = "source.apiDefinitionV4.endpointGroups")
+    @Mapping(target = "flowExecution", source = "source.apiDefinitionV4.flowExecution")
+    @Mapping(target = "flows", source = "source.apiDefinitionV4.flows")
+    @Mapping(target = "lifecycleState", source = "source.apiLifecycleState")
+    @Mapping(target = "links", expression = "java(computeCoreApiLinks(source, uriInfo))")
+    @Mapping(target = "listeners", source = "source.apiDefinitionV4.listeners", qualifiedByName = "fromListeners")
+    @Mapping(target = "state", source = "source.lifecycleState")
+    ApiV4 mapToV4(io.gravitee.apim.core.api.model.Api source, UriInfo uriInfo, GenericApi.DeploymentStateEnum deploymentState);
```

Comment:
It doesn't look like `deploymentState` is being used in the mapper?

Reply:
Good catch I've copy paste without thinking 🙈